.github/workflows/workflow.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
       - run: yarn build
         env:
           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-          SENTRY_ENABLED: 'true'
+          # SENTRY_ENABLED: 'true'
```

Comment:
Blir denne retta på i ein anna PR?

Reply:
måtte disable den for å få bygget til å kjøre, tenkte å re-enable så fort sentry ikke fjusker lengre